daemon/seccomp_linux.go

```diff
@@ -3,17 +3,16 @@
 package daemon
 
 import (
-	"encoding/json"
 	"fmt"
 
 	"github.com/Sirupsen/logrus"
 	"github.com/docker/docker/container"
-	"github.com/docker/engine-api/types"
+	"github.com/docker/docker/profiles/seccomp"
```

Comment:
The conversion for engine-api definition to specs definition now just happens inside `profiles/seccomp` package. Loadable profile is still using the engine-api type.